python_modules/dagster/dagster/components/scaffold/scaffold.py

```diff
@@ -101,19 +101,27 @@ class ScaffoldRequest(Generic[TModel]):
 
 
 @public
-@preview(emit_runtime_warning=False)
 class Scaffolder(Generic[TModel]):
-    """Handles scaffolding its associated scaffold target."""
+    """Handles scaffolding its associated scaffold target based on user-supplied parameters.
+    Invoked by a user using the `dg scaffold` CLI command.
+
+    To associate a scaffolder with its target class, use the :py:func:`scaffold_with` decorator.
```

Comment:
does the `:py:func:` thing work without specifying `dagster.scaffold_with`? genuinely not sure